packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -953,6 +955,20 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 		return columnFilterValue;
 	}
 
+	/**
+	 * Get the position of the filter box of the specified column.
+	 * @param columnName - Name of the column whose filter box position should be retrieved.
+	 * @returns The position of the filter box of the specified column.
+	 */
+	public getColumnFilterPosition(columnName: string): StarkTableColumnFilter["filterPosition"] | undefined {
+		if (!(this.filter.columns instanceof Array)) {
+			return;
+		}
+
+		const column = this.filter.columns.find((columnFilter: StarkTableColumnFilter) => columnFilter.columnName === columnName);
+		return column && column.filterPosition;
```

Comment:
What is returned here if there is no column? `null`? `undefined`? 

I think we should return something explicitly: `undefined`

Reply:
yes that's right, I've adapted the code

Reply:
Does it mean we need a new if case here ?

Reply:
Mmm yes, but you can do it in the same line like this:

`return column ? column.filterPosition : undefined;`

Reply:
hm, ok I see.

It's changed :-)